CONTRIBUTING.md

```diff
@@ -47,6 +47,30 @@ paper and understand how it fits into the TensorFlow community. This
 process can take longer than typical commit reviews so please bare with
 us**
 
+## Colab Support
```

Comment:
I would name this section "Contributing Examples" And then add a small blurb about how all examples are built using colab and then saved as .ipynb files in the repo